src/ert/_c_wrappers/enkf/ert_config.py

```diff
@@ -28,10 +27,11 @@
 from ert._c_wrappers.util import SubstitutionList
 from ert._clib import job_kw
 from ert._clib.config_keywords import init_site_config_parser, init_user_config_parser
+from ert.parsing import ConfigValidationError, ConfigWarning
+from ert.parsing.lark_parser import parse as lark_parse
```

Comment:
This should probably be exported through `ert.parsing` ie. `from ert.parsing import parse as lark_parse`, and `ert.parsing.lark_parse` be named `ert.parsing._lark_parser` so to be explicitly private.

Reply:
Done & done, also moved declarations from __init__ to a commons file to avoid circular imports, as __init__ exporting lark_parser and lark_parser importing from __init__.